### PR TITLE
Detail on changing primary storage scope added to adminguide and concepts

### PR DESCRIPTION
**Pull request by abh1sar**
Documentation for Feature PR : https://github.com/apache/cloudstack/pull/8875

----
📚 Documentation preview 📚: https://cloudstack-documentation--395.org.readthedocs.build/en/395/

### CODE REVIEW

**Review by weizhouapache — APPROVED** [review]
lgtm
https://cloudstack-documentation--395.org.readthedocs.build/en/395/adminguide/storage.html#changing-the-scope-of-the-primary-storage

**Comment by weizhouapache** [comment]
@abh1sar 
can you add a screenshot of cluster->zone change ?

**Comment by abh1sar** [comment]
> @abh1sar can you add a screenshot of cluster->zone change ?

@weizhouapache you mean the cluster->zone change scope form in the adminguide?

 
clusterToZone

**Comment by abh1sar** [comment]
@andrijapanicsb please review

**Comment by weizhouapache** [comment]
> > @abh1sar can you add a screenshot of cluster->zone change ?
> 
> @weizhouapache you mean the cluster->zone change scope form in the adminguide?
> 
> clusterToZone

@abh1sar 
looks good, can you add this image as well ?

**Comment by weizhouapache** [comment]
@abh1sar 
there are conflicts

**Comment by abh1sar** [comment]
@weizhouapache resolved conflicts and added Cluster->Zone UI screenshot

File: source/_static/images/change-storage-pool-scope-to-zone.png
Comment (shwstppr) [inline source/_static/images/change-storage-pool-scope-to-zone.png:1]: Do we need to update this image if the dialog has changed?

**Review by shwstppr — APPROVED** [review]
Changes LGTM

**Comment by andrijapanicsb** [comment]
@blueorangutan docbuild

**Comment by andrijapanicsb** [comment]
I don't see the other image, but I've asked BO to build docs again, and if both images are visible, I'm LGTM with the rest, nice explanation.

**Review by andrijapanicsb — APPROVED** [review]
LGTM

File: source/adminguide/storage.rst [hunk 1]
```diff
@@ -185,6 +185,38 @@ ensure that the protocol is set to "Filesystem".
...
+
+|change-storage-pool-scope-to-zone.png|
+
+Scope change from Zone to Cluster will disconnect the Primary Storage from all
+Hosts that were previously connected to the Primary Storage and are not a part
+of the specified Cluster. So, if there are running VMs on such hosts using this
+Storage Pool, it cannot be disconnected from the hosts. In this case the Scope
+change operation will error out. Such VMs need to be stopped or migrated first.
+
+This feature is tested and supported for the following Hypervisor-Protocol-
```
Comment (andrijapanicsb) [inline source/adminguide/storage.rst:207]:
```suggestion
This feature is tested and supported for the following Hypervisor and storage combinations
```

File: source/adminguide/storage.rst [hunk 2]
```diff
@@ -185,6 +185,38 @@ ensure that the protocol is set to "Filesystem".
...
+Storage Provider combinations:
```
Comment (andrijapanicsb) [inline source/adminguide/storage.rst:208]:
```suggestion

```

File: source/adminguide/storage.rst [hunk 3]
```diff
@@ -185,6 +185,38 @@ ensure that the protocol is set to "Filesystem".
...
+
+-  KVM - NFS - Default Primary
```
Comment (andrijapanicsb) [inline source/adminguide/storage.rst:210]:
```suggestion
-  KVM with NFS
```

File: source/adminguide/storage.rst [hunk 4]
```diff
@@ -185,6 +185,38 @@ ensure that the protocol is set to "Filesystem".
...
+
+-  KVM - CEPH/RBD - Default Primary
+
+-  VMWare - NFS - Default Primary
```
Comment (andrijapanicsb) [inline source/adminguide/storage.rst:214]:
```suggestion
-  VMWare with NFS
```

**Comment by andrijapanicsb** [comment]
I made a few proposals to make it more readable from the end user (user doesn't necessarily care about "protocol" and "provider" - simply saying "KVM with NFS" is enough - again - just a proposal from my side, to simplify it.

File: source/_static/images/change-storage-pool-scope-to-zone.png
Comment (abh1sar) [reply source/_static/images/change-storage-pool-scope-to-zone.png:1]: Yes, updated the image.

File: source/adminguide/storage.rst (hunk 1 above)
Comment (abh1sar) [reply source/adminguide/storage.rst]: Done

File: source/adminguide/storage.rst (hunk 3 above)
Comment (abh1sar) [reply source/adminguide/storage.rst]: Done

File: source/adminguide/storage.rst [hunk 5]
```diff
@@ -185,6 +185,38 @@ ensure that the protocol is set to "Filesystem".
...
+of the specified Cluster. So, if there are running VMs on such hosts using this
+Storage Pool, it cannot be disconnected from the hosts. In this case the Scope
+change operation will error out. Such VMs need to be stopped or migrated first.
+
+This feature is tested and supported for the following Hypervisor-Protocol-
+Storage Provider combinations:
+
+-  KVM - NFS - Default Primary
+
+-  KVM - CEPH/RBD - Default Primary
```
[the remark this answers is not in the document]
Comment (abh1sar) [reply source/adminguide/storage.rst]: Done

File: source/adminguide/storage.rst (hunk 4 above)
Comment (abh1sar) [reply source/adminguide/storage.rst]: Done

**Comment by abh1sar** [comment]
@blueorangutan docbuild

**Comment by abh1sar** [comment]
@blueorangutan docbuild

**Comment by abh1sar** [comment]
@blueorangutan docbuild

**Comment by abh1sar** [comment]
@blueorangutan docbuild

**Pull request merged**